tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -1378,3 +1383,31 @@ Then(
     assert.ok(collaboratorsArray.indexOf(lastLink.token) > -1)
   }
 )
+
+Given('user {string} has locked file/folder {string} setting following properties', function(
+  userId,
+  fileName,
+  table
+) {
+  const properties = table.rowsHash()
+  return webdav.lockResource(userId, fileName, properties)
+})
+
+When('the user closes rename dialog', function() {
+  return client.page.FilesPageElement.filesList().closeRenameDialog()
+})
+
+Then('notifications should be displayed on the webUI with the text', async function(message) {
+  const actualMessages = await client.page.webPage().getPopupErrorMessages()
+  const isMessageShown = assertIncludesMessage(actualMessages, message)
```

Comment:
you use assert.containsText (https://nightwatchjs.org/api/#assert-containsText) here

Reply:
yes but when we want to check for multiple files/foldler on same scenario then that would fail because containsText will get  only the text of first popup (multiple popup messages are displayed)

Reply:
Yeah, but if we check for multiple files like in here
https://github.com/owncloud/web/blob/6d42b4f489d535648a58e4bab2b4e792ebda8143/tests/acceptance/features/webUIWebdavLocks/locks.feature#L339-L347
then there will be multiple popups and containsText will only get the text of the first popup